optaplanner-operator/src/main/java/org/optaplanner/operator/impl/solver/model/SolverStatus.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.operator.impl.solver.model;
+
+public final class SolverStatus {
```

Comment:
Let's not name a class SolverStatus, that name is already used by org.optaplanner.core.api.solver.SolverStatus

Reply:
Let's solve the naming in the previous comment; the naming scheme will apply here, too.